composer.json

```diff
@@ -11,8 +11,9 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "ezsystems/ezpublish-legacy": ">=2014.11",
-        "ezsystems/ezpublish-kernel": "~6.0@dev",
+        "ezsystems/ezpublish-legacy-installer": "^2.0.4",
+        "ezsystems/ezpublish-legacy": ">=2017.08",
```

Comment:
Nope, that was what I first called it, but if we aim for parity with the features in the milestone, then that needs work also on platform side, hence 1.11 => august => 08 :)